server/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Server implementation for CampusUnify",
   "main": "app.js",
   "scripts": {
-    "start": "node server.js",
+    "start": "nodemon server.js",
```

Comment:
Please don't change the package scripts

Reply:
Tha package script was changed, because with nodemon the server starts again automatically when changed are made in the code.

Reply:
You are supposed to run the server with `npm run dev` rather than `npm start` for the development environment. The npm start is meant for the production environment.